app/main.py

```diff
@@ -1,8 +1,51 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: float,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
```

Comment:
![](https://user-images.githubusercontent.com/117740672/255618922-55bea0dd-5963-4927-b8fb-03ee25cd3fd2.png)

Reply:
Should I do it like in the screenshot (each argument on a new line) or how?